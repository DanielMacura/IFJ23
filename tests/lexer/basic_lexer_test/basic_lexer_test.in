write("Zadejte cislo pro vypocet faktorialu: ")
let inp : Int? = readInt()