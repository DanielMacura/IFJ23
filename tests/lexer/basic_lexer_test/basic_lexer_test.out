IDENTIFIER
LBRACKET
STRING:Zadejte\032cislo\032pro\032vypocet\032faktorialu:\032
RBRACKET
EOL
KW_LET
VARIABLE
COLON
KW_INT
QUESTIONMARK
EQUALS
IDENTIFIER
LBRACKET
RBRACKET
EOF
